.changeset/tough-hats-draw.md

```diff
@@ -0,0 +1,54 @@
+---
+"@salt-ds/core": patch
+---
+
+Refactor internal implementation of button styles
+
+- this is a non-breaking refactor of how we organise the styles within the CSS file.
```

Comment:
checking, is this definitely considered non breaking/..only a patch? I personally have used a lot of --saltComponent tokens myself which would have to be changed by this